README.md

```diff
@@ -462,35 +464,52 @@ The `uf2create` task is used to generated an uf2 binary suitable for running on
 
     Use this plugin to create Raspberry Pico uf2 files from an AtomVM packbeam file.
 
-    Usage: rebar3 atomvm uf2create [-o <output>] [-s <start>] [-i <input>]
+    Usage: rebar3 atomvm uf2create [-u <family_id>] [-o <output>]
+                               [-a <start>] [-i <input>]
 
-    -o, --output  Output path/name
-    -s, --start   Start address for the uf2 binary (default 0x10180000)
-    -i, --input   Input avm file to convert to uf2
+      -f, --family_id  Flavor of uf2 file to create (default rp2040)
+      -o, --output      Output path/name
+      -s, --start       Start address for the uf2 binary (default 0x10180000)
+      -i, --input       Input avm file to convert to uf2
 
 It should not be necessary to use this tool before using `pico_flash`, unless you have built a custom VM that requires changing the start address of the uf2 binary. If the application has not been compiled, or packed with packbeam, these steps will be run first using the default settings for `packbeam`.
 
+The following table enumerates the valid `family_id` options:
+
+| Key         | Description |
+|-------------|-------------|
+| `rp2040`    | Original Raspberry Pi Pico and Pico-W |
+| `rp2035`    | Raspberry Pi Pico 2 |
+| `data`      | Raspberry Pi Pico 2 |
+| `universal` | Universal format for both `rp2040` and `rp2035` |
+
+> Note the convenience of universal uf2 binaries comes with the expense of being twice the size, as both versions are included in the universal uf2.
+
 The following table enumerates the properties that may be defined in your project's `rebar.config` file for this task.  Use `uf2create` as the key for any properties defined for this task.
 
 | Key | Type | Description |
 |-----|------|-------------|
 | `start` | `string()` | Start address for the uf2 binary |
+| `family_id` | `string()` | The family\_id or flavor of uf2 to build |
 
 Example:
 
-    {atomvm_rebar3_plugin, [{uf2create, [{start, "0x10180000"}]}]}.
+    {atomvm_rebar3_plugin, [{uf2create, [{app_id, "0x10180000"},{family_id,universal}]}]}.
```

Comment:
app_id ?

Reply:
Not sure what happened there... I had started to us `app_start`, thinking `start` could be passed to the flash task, and it would be passed down to the packbeam task, but that is not the case at the moment, so I didn't want to make an unnecessary breaking change. It's fixed now anyway.

In a future PR I would like to make the change, that any option not used my the called task is passed to dependent tasks, but this will require breaking changes to several tasks to remove option name, and short option conflicts.